pkg/apis/kudo/v1beta1/instance_validator.go

```diff
@@ -15,17 +15,17 @@ var _ kudo.Validator = &Instance{}
 
 // ValidateCreate implements kudo.Validator (slightly tweaked interface originally from controller-runtime)
 // we do not enforce any rules upon creation right now
-func (i *Instance) ValidateCreate(req admission.Request) error {
+func (in *Instance) ValidateCreate(req admission.Request) error {
```

Comment:
Nööö.. `i` is not ideal but we got used to it being an *Instance*. `in` sounds too much link *Input*.

Reply:
> > * Maybe it would make sense to move the full reconcile method from the controller to the operation file.
> 
> I'd be against it. Readers (myself included) expect to find main controller loop in `/controllers/instance/instance_controller.go` having coarse-grained logical steps towards state reconciliation. All the helper methods could live someplace else.

Yeah, i'm not too fond of this either. I'll try moving this around just inside the package, in some separate files but without the extra struct, let's see how it looks then.